docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -165,11 +165,112 @@ Upgrading from Chef Backend 1.x to Chef Backend 2.x requires full cluster downti
 
 | Running Version | Upgrade To Version | Upgrade Type |
 |---------|---------|------|
-| 2.x | 2.x | Rolling |
-| 2.1 | 3.0 | Rolling |
-| 2.0 | 3.0 | Downtime |
+| 2.0 | 2.1 | Rolling |
+| 2.1 | 3.0 | Rolling if the created string in curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human is 5.6.16 or greater|
+| 2.1 | 3.0 | Downtime on the FrontEnds if the created string in curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human is 5.4.1 or lesser.|
+
+Chef Backend clusters that were originally installed with a version of Chef Backend before 2.1.0 will need to delete and reindex elasticsearch indices after upgrading.
+
+To determine if your indexes need to be replaced run the command:
+
+```bash
+curl -XGET localhost:9200/chef/_settings/index.version.created*?pretty\&human
+```
+
+and look for the version number in "created_string"
+
+If "created_string" : "2.3.1" or any version less than "5.6.16" you will need to perform these additional steps:
+
+1. Run this command on all front end nodes to stop them. Alternatively put them in maintenance modei(disable the API temporarily) if running Infra Server 14.6.32 or later.
+   ```bash
+   sudo chef-server-ctl stop
+   ```
+   ```bash
+   sudo chef-server-ctl maintenance on
+   ```
+
+2. Identify the node with the **leader** role using the `chef-backend-ctl cluster-status` command:
+
+    ```none
+    Name       IP              GUID                              Role      PG        ES
+    backend-1  192.168.33.215  dc0c6ea77a751f94037cd950e8451fa3  leader    leader    not_master
+    backend-2  192.168.33.216  008782c59d3628b6bb7f43556ac0c66c  follower  follower  not_master
+    backend-3  192.168.33.217  1af654172b1830927a571d9a5ba7965b  follower  follower  master
+    ```
+
+    In this example, `backend-1` is the **leader** node, as indicated by its role in the **Role** column.
+
+3. Install the new Chef Backend package on all nodes in the cluster:
+
+    RHEL and CentOS:
+
+    ```bash
+    yum install PATH_TO_FILE.rpm
+    ```
+
+    Debian and Ubuntu:
+
+    ```bash
+    dpkg -i PATH_TO_FILE.deb
+    ```
+
+4. Upgrade on follower nodes first:
+
+    ```bash
+    chef-backend-ctl upgrade
+    ```
+
+5. Upgrade the leader node with the `--failover` option:
+
+    ```bash
+    chef-backend-ctl upgrade --failover
+    ```
+
+6. Execute each of these commands on all Chef Backend nodes before proceeding to the next command:
```

Comment:
Add context around why we are doing this at the end. This will create indexes for 6.8.23 instead of an older version that we are starting the rolling upgrade from.